modules/4337/test/erc4337/ERC4337ModuleExisting.spec.ts

```diff
@@ -119,16 +140,18 @@ describe('Safe4337Module - Existing Safe', () => {
       const safeOpHash = calculateSafeOperationHash(await validator.getAddress(), safeOp, await chainId())
       const signature = buildSignatureBytes([await signHash(user1, safeOpHash)])
       const userOp = buildUserOperationFromSafeUserOperation({ safeOp, signature })
-
-      const transaction = await entryPoint.executeUserOp(userOp, ethers.parseEther('0.000001')).then((tx) => tx.wait())
-      const logs = transaction.logs.map((log) => entryPoint.interface.parseLog(log))
-      const emittedRevert = logs.some((l) => l?.name === 'UserOpReverted')
-
-      expect(emittedRevert).to.be.true
+      const userOpHash = await entryPoint.getUserOpHash(userOp)
+      // Error('Execution failed')
+      const expectedReturnData =
+        '0x08c379a000000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000010457865637574696f6e206661696c656400000000000000000000000000000000'
```

Comment:
nit: This should be doable with Ethers:
```suggestion
      const expectedReturnData = validator.interface.encodeErrorResult('Error(string)', 'Execution failed')
```